services/src/test/scala/cromwell/services/database/LobSpec.scala

```diff
@@ -61,19 +61,7 @@ class LobSpec extends FlatSpec with Matchers with ScalaFutures {
 
       val workflowStoreEntries = Seq(workflowStoreEntry)
 
-      val future = databaseSystem.platform match {
-        case MysqlDatabasePlatform =>
-          // MySQL crashes because it calls SerialBlob's getBytes instead of getBinaryStream
-          database.addWorkflowStoreEntries(workflowStoreEntries).failed map { exception =>
-            exception should be(a[SerialException])
-            exception.getMessage should
-              be("Invalid arguments: position cannot be less than 1 or greater than the length of the SerialBlob")
-          }
-        case _ =>
-          database.addWorkflowStoreEntries(workflowStoreEntries)
-      }
-
-      future.futureValue
+      database.addWorkflowStoreEntries(workflowStoreEntries).futureValue
```

Comment:
Why don't we need special handling for MySql anymore?

Reply:
presumably because the upgraded dependency corrected the problem.